bundles/core/src/test/java/com/adobe/cq/email/core/components/internal/models/ContainerImplTest.java

```diff
@@ -40,52 +40,52 @@
 import static org.junit.jupiter.api.Assertions.assertNull;
 
 @ExtendWith({AemContextExtension.class, MockitoExtension.class})
-class ContainerModelTest {
+class ContainerImplTest {
 
     private final AemContext ctx = new AemContext();
-    private ContainerModel underTest;
+    private ContainerImpl underTest;
 
     @BeforeEach
     void setUp() {
-        ctx.addModelsForClasses(ContainerModel.class);
+        ctx.addModelsForClasses(ContainerImpl.class);
```

Comment:
I would normally use the API/interface instead of the impl. Same for all the `adaptTo(..)` calls.